backend/src/apiserver/config.go

```diff
@@ -23,7 +24,9 @@ import (
 )
 
 func initConfig() {
-	// Import environment variable
+	// Import environment variable, support nested vars e.g. OBJECTSTORECONFIG_ACCESSKEY
+	replacer := strings.NewReplacer(".", "_")
```

Comment:
Why do we need to replace dots (given that the environment variables can use underscores directly)?

Reply:
@Ark-kun its the other way around, the config file uses dots (nested), we convert the `_` to `.` so it will work with Viper e.g. `ObjectStoreConfig.AccessKey` can be overwritten by `OBJECTSTORECONFIG_ACCESSKEY` env var if it exists

see more in: https://github.com/spf13/viper/issues/239

example env vars definition:
```
        - name: OBJECTSTORECONFIG_ACCESSKEY
          value: minio
        - name: OBJECTSTORECONFIG_SECRETACCESSKEY
          value: minio123
        - name: OBJECTSTORECONFIG_BUCKETNAME
          value: mlpipeline
        - name: OBJECTSTORECONFIG_PORT
          value: "9000"
        - name: OBJECTSTORECONFIG_HOST
          value: minio-service
```